app/assets/v2/js/grants/fund.js

```diff
@@ -516,6 +516,8 @@ const predictCLRMatch = () => {
 
   const contributions_axis = [ 0, 1, 10, 100, 1000, 10000 ];
 
+  let index = 0
```

Comment:
lol i meant just replace index with 0 :P

Reply:
yeah but its used on lines 524 and 525